compiler/src/dmd/expressionsem.d

```diff
@@ -14775,6 +14775,8 @@ private bool checkArithmetic(Expression e, EXP op)
         // Deprecated in 2.111
         // In 2.121, remove this branch to let `checkValue` raise the error
         deprecation(e.loc, "type `%s` has no value", e.toChars);
+        if (!e.type.isOpaqueType)
+            deprecationSupplemental(e.loc, "perhaps use `%s`.init", e.toChars);
```

Comment:
backtick in the wrong place , you want
```
`%s.init`
```

Reply:
I'm getting sleepy as well :slightly_smiling_face: